scripts/kg_build.py

```
@@ -490,50 +651,99 @@ def _execute_pipeline(
+        """,
+        {"source_path": str(source_path), "document_name": source_path.name},
+        database_=database,
+    )
+
+
 def run(argv: Sequence[str] | None = None) -> dict[str, Any]:
     """
     Builds a knowledge graph from a source file and writes a structured JSON run log.
     
-    Parses CLI arguments (or uses provided argv), validates environment and chunking parameters, runs the SimpleKGPipeline to ingest the source text into Neo4j using OpenAI-based embedder and LLM adapters, collects database counts, writes a sanitized JSON log to disk, and returns the log dictionary.
+    Parses CLI arguments (or uses provided argv), validates environment and chunking parameters, ingests the source text into Neo4j using configured OpenAI clients and the pipeline, ensures document–chunk relationships, collects database counts, writes a sanitized JSON log to disk, prints the log, and returns the log dictionary.
```

Ruff (0.13.1)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)